app/javascript/components/App.js

```diff
@@ -26,13 +26,14 @@ class App extends Component {
       <Router>
       <Header {...this.props} />
       <Switch>
-        <Route exact path="/" component={Home} />
+        <Route exact path="/"> <Home /> </Route>
 
-        <Route path="/gooseynew" component={GooseyNew} />
 
-        <Route path="/gooseyedit" component={GooseyEdit} />
+        {/* <Route path="/gooseynew"> <GooseyNew />  </Route> */}
```

Comment:
Why are these commented out?

Reply:
We are going to uncomment them as we create those pages.

Reply:
For future, best practice is to remove commented out code before merging to main.